### PR TITLE
Update branding to 10.0

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TieredCompilation>false</TieredCompilation>
     <DefineConstants>$(DefineConstants);IS_BENCHMARKS</DefineConstants>
+    <SkipMicrobenchmarksValidation>true</SkipMicrobenchmarksValidation>
```
Comment (JamesNK) [inline src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj:9]: What is this change for?

Comment (wtgodbe) [reply src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj:9]: The microbenchmark fails to find the just built SharedFx - this happens every time we update the branding/TFM in certain microbenchmarks, we're able to re-enable it once we get an SDK with the new TFM

Comment (JamesNK) [reply src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj:9]: Ok. Maybe add a comment?

**Review by JamesNK — APPROVED** [review]

File: src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj (hunk 1 above)
Comment (wtgodbe) [reply src/Grpc/JsonTranscoding/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj:9]: https://github.com/dotnet/aspnetcore/issues/57461

**Pull request merged**